developers/contributor-guide/weaviate-modules/how-to-build-a-new-module.md

```diff
@@ -146,4 +146,4 @@ tools/dev/run_dev_server.sh --<LOCAL-CONFIG-NAME>
 You can now load in any sample or test dataset. If you only make changes in the `/modules/<new-module>` folder afterwards, you only need to re-run `tools/dev/run_dev_server.sh --<LOCAL-CONFIG-NAME>` to apply the changes. The data will be kept, so no need to re-import.
 
 #### Passing tests
-Finally, and before you make a PR, your Weaviate with your complete module implementation should pass all tests (all existing tests and tests added by you). How to run tests, [check this page](../weaviate-core/tests.md#run-the-whole-pipeline).
+Finally, and before you make a PR, your Weaviate with your complete module implementation should pass all tests (all existing tests added by you). How to run tests, [check this page](../weaviate-core/tests.md#run-the-whole-pipeline).
```

Comment:
The "and" is needed here. Consider: 

Before you submit your PR, your new module implementation must pass all existing tests and any new tests that you added. 

For details on how to run tests, see [Run the whole pipeline](../weaviate-core/tests.md#run-the-whole-pipeline).